Assets/Scripts/R&D Tree/RDTreeManager.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class RDTreeManager : MonoBehaviour
+{
+    [SerializeField] private GameObject _skillTree;
+
+    private RDControls _controls;
+
+    public static RDTreeManager Instance;
+
+    public ResearchNode CurrentResearching;
```

Comment:
Try not to use any public variables in a class. Instead use getters and setters to control the value of the variable